src/utils.ts

```diff
@@ -45,6 +43,7 @@ export function parseYAML(
   }
   return {name, author, description, using}
 }
+const removeGreaterLessThan = (item:string) => item.replace(/\>/g,'&#62;').replace(/\</g,'&#60;')
```

Comment:
You are now bypassing the `sanitize` method completely, which I added for a reason :-D. Can we embed this code into the sanitize function and revert to calling that code again?